deployment-examples/gcp/extension/k8s/values.yaml

```diff
@@ -17,12 +17,3 @@ ingress:
   annotations:
     kubernetes.io/ingress.class: 'gce'
     kubernetes.io/ingress.allow-http: 'true'
-
```

Comment:
Why it is removed? Shouldn't we use our health endpoint? Otherwise, there is no need to have health endpoint from the application.

Reply:
It's the default probe url in the helm chart, I also added the link, you might check.